tests/config/functional/ini/pylintrc_with_multi_line_init_hook.ini

```diff
@@ -0,0 +1,6 @@
+# Reported in https://github.com/PyCQA/pylint/issues/6888
```

Comment:
Shouldn't the result.json change ? Probably not as init_hook are treated as a special case before the config is instanced, so we should probably create a unit test with multiple print so we can actually check that the code of each line was executed.

Reply:
This raises a `SyntaxError` and fails the test without the changes in the PR. So it is actually a test against the reported issue: multi-line init hooks became syntactically incorrect code.